src/test/java/org/gridsuite/gateway/ElementAccessControlTest.java

```diff
@@ -15,6 +15,7 @@
 import com.nimbusds.jose.jwk.gen.RSAKeyGenerator;
 import com.nimbusds.jwt.JWTClaimsSet;
 import com.nimbusds.jwt.SignedJWT;
+import org.gridsuite.gateway.config.ServiceURIsConfig;
```

Comment:
Lonely import addition seems useless to me. I guess it passed the lint step because it is test code.

Reply:
the package has been changed to org.gridsuite.gateway.config it was  org.gridsuite.gateway